hadoop-connector-examples/README.md

```diff
@@ -101,3 +101,45 @@ You can also use hadoop-connectors on Spark
 Spark (verified with Spark 2.2.1 on Ubuntu 16.04)
    spark-submit --class io.pravega.examples.spark.WordCount ${HADOOP_EXAMPLES_JAR} ${PRAVEGA_URI} ${PRAVEGA_SCOPE} ${PRAVEGA_STREAM}
 ```
+
+## Terasort
+
+Hadoop (verified with Hadoop 2.8.1 and 3.1.1 on Ubuntu 16.04)
+
+### Execution
+
+```
+1. setup and start hdfs as well as pravega
+
+2. set env variables
+   export HDFS=hdfs://<hdfs_ip_and_port> # e.g. hdfs://192.168.0.188:9000
+   export HADOOP_EXAMPLES_JAR=<pravega-hadoop-examples-x.y.z-SNAPSHOT-all.jar location> # e.g. ./build/libs/pravega-hadoop-examples-0.4.0-SNAPSHOT-all.jar
+   export HADOOP_EXAMPLES_INPUT_DUMMY=${HDFS}/tmp/hadoop_examples_input_dummy
+   export HADOOP_EXAMPLES_OUTPUT=${HDFS}/tmp/hadoop_examples_output
+   export PRAVEGA_URI=tcp://<pravega_controller_ip_and_port> # e.g. tcp://192.168.0.188:9090
+   export PRAVEGA_SCOPE=<scope_name>   # e.g. myScope
+   export PRAVEGA_INPUT_STREAM=<stream_name> # e.g. intStream
+   export PRAVEGA_OUTPUT_STREAM=<stream_name> # e.g. outStream
+   export PRAVEGA_OUTPUT_STREAM_PREFIX=<stream_name_prefix> # e.g. sortedStream
+   export CMD=wordcount # so far, can also try wordmean and wordmedian
+
+3. make sure input and output dir are empty
+   hdfs dfs -rmr ${HADOOP_EXAMPLES_INPUT_DUMMY}
+   hdfs dfs -rmr ${HADOOP_EXAMPLES_OUTPUT}
+
+4. run hadoop commands
+
+   4.1 to generate events into a Pravega stream
+      hadoop jar ${HADOOP_EXAMPLES_JAR} teragen [1m|1b|1t] ${HADOOP_EXAMPLES_INPUT_DUMMY} ${PRAVEGA_URI} ${PRAVEGA_SCOPE} ${PRAVEGA_STREAM}
+      e.g. hadoop jar $HADOOP_EXAMPLES_JAR teragen 10 hdfs://localhost:8020/pravega/input tcp://localhost:9090 myScope inputStream
```

Comment:
The comment says 
> hadoop jar ${HADOOP_EXAMPLES_JAR} teragen [1m|1b|1t] ${HADOOP_EXAMPLES_INPUT_DUMMY} ${PRAVEGA_URI} ${PRAVEGA_SCOPE} ${PRAVEGA_STREAM}

but the sample is provided as 

> e.g. hadoop jar $HADOOP_EXAMPLES_JAR teragen 10 hdfs://localhost:8020/pravega/input tcp://localhost:9090 myScope inputStream

do we need to pass absolute byte value or it can take only these `[1m|1b|1t]` values?

Reply:
It only takes number of line, and each line is 100 bytes